draft-ietf-add-split-horizon-authority.xml

```diff
@@ -386,7 +389,8 @@
           <li><strong>Insecure</strong>: The client <bcp14>SHOULD</bcp14> retry the validation
             process using a different method, such as the one in <xref
             target="validating-external"/>, to ensure compatibility with
-            unsigned names.</li>
+            unsigned names. If the client chooses not to retry, it MUST consider
```

Comment:
```suggestion
            unsigned names. If the client chooses not to retry (e.g., no configured policy to send queries to an encrypted DNS resolver outside the network), it MUST consider
```

Reply:
If the client has not configured a encrypted DNS resolver outside of the network, this draft is not required (it will discover and use the locally discovered encrypted resolver).

Reply:
I know.

This is the only example I found to exemplify why the client chooses to not retry here. If you have another example please add it. Thanks.

Reply:
we could say something like this: (e.g., no configured policy to validate the authorization claim using an external resolver). I mean the client has a pre-configured external resolver but wants to use it only for resolving DNS queries but not to validate the authorization claims.

Reply:
Works for me. Thanks.